android/build.gradle

```
@@ -6,6 +6,7 @@ buildscript {
         minSdkVersion = 21
         compileSdkVersion = 30
         targetSdkVersion = 30
+        kotlinVersion = '1.3.41'
```

빌드할 때 Kotlin 버전 문제로 `react-native-screens`에서 에러를 띄우는 문제가 있습니다.
버전만 잘 바꿔주면 큰 문제 없이 동작하니 수정 부탁드려요!
```suggestion
        kotlinVersion = '1.4.10'
```

---

확인 감사합니다. 🙇‍♀️